src/pages/patientView/mutation/column/AlleleFreqColumnFormatter.tsx

```diff
@@ -98,10 +98,9 @@ export default class AlleleFreqColumnFormatter {
         }
 
         // as long as we have tooltip lines, show tooltip in either cases (single or multiple)
-        if (tooltipLines.length > 0)
+        if (tooltipLines.length > 0 && freqs.filter((freq) => freq !== undefined).length > 0)
```

Comment:
optimization: use _.some when you are checking for "at least one" because it will stop iteration after it find a single element that complies with condition.